tools/ARIAtools/logger.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Author: Rohan Weeden
+# Copyright 2020, by the California Institute of Technology. ALL RIGHTS
+# RESERVED. United States Government Sponsorship acknowledged.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+Global logging configuration
+"""
+import logging
+import os
+import sys
+from logging import FileHandler, Formatter, StreamHandler
+
+
+# Inspired by
+# https://stackoverflow.com/questions/384076/how-can-i-color-python-logging-output
+class UnixColorFormatter(Formatter):
+    yellow = "\x1b[33;21m"
+    red = "\x1b[31;21m"
+    bold_red = "\x1b[31;1m"
+    reset = "\x1b[0m"
+
+    COLORS = {
+        logging.WARNING: yellow,
+        logging.ERROR: red,
+        logging.CRITICAL: bold_red
+    }
+
+    def __init__(self, fmt=None, datefmt=None, style="%", use_color=True):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5957665)